listener/listener.go

```diff
@@ -320,8 +320,13 @@ func ensureSubscription(ctx context.Context, sm *servicebus.SubscriptionManager,
 	if err == nil {
 		return subEntity, nil
 	}
-
-	return sm.Put(ctx, name)
+	increaseMaxDelivery := func(s *servicebus.SubscriptionDescription) error {
+		var maxDeliveryCount int32 = 500
```

Comment:
Do we increase the delivery count for every message? We have a few cases to abandon messages. If the answer is yes, does it mean that every message will be abandoned much more times?

Reply:
Yes this means messages that go through retry laters will be aboned/delivered several times. And if we go over 500*5 minutes then we give up but I think that's okay as long as its higher than when regional looper will give us a new message.